app/build.gradle

```diff
@@ -36,6 +36,7 @@ android {
     }
     buildTypes {
         debug {
+            pseudoLocalesEnabled false
```

Comment:
When set to `true`, this indicates that our app supports pseudolocales (https://developer.android.com/guide/topics/resources/pseudolocales.html). The default value is `false`, so adding this line makes no functional difference. It just makes it easier to enable it when it is desired.

We could probably leave this enabled all the time, and it won't affect anything day to day; not unless we're doing locale-related work and meet the following criteria. But to be safe, I'll leave it set to `false` so I don't change any default behaviour.

Pseuodolocales will show if all of the following criteria are met:

1. app is in `DEBUG` mode
2. developer settings are enabled
3. user has selected a pseuolocale from their device language settings